### PR TITLE
Bulk actions print

### PR DESCRIPTION
**Pull request by cantsin**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/734)

## What does this change?

- Adds a new template for printing complaint details. the old complaint details page was problematic because
    1. form duplication resulted in errors in the console regarding duplicate ids and hidden named inputs
    2. a lot of data was being unnecessarily transferred over for each report
- Undoes previous print changes for complaint details
- Tries to re-use existing single print functionality where-ever possible
- Refactors refactoring query in views 
- Adds the printer svg icon (previously we were using the fax svg icon)

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by cantsin** [comment]
@Jkrzy This PR may be over the place since there's refactoring involved; I can split it up in multiple PRs if you would prefer.

File: crt_portal/cts_forms/views.py
Comment (cantsin) [inline crt_portal/cts_forms/views.py:313]: I'm not sure why this was set as a array with a single item, so let me know if I'm missing anything here. (As far as I can see, we only reference this variable once, as `{{primary_complaint.0}}` in the template.)

File: crt_portal/cts_forms/models.py
Comment (Jkrzy) [inline crt_portal/cts_forms/models.py:321]: Django has a built-in method we can leverage to provide this functionality since we're specifying a `choices` attribute on the model field! 

https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.get_FOO_display

`get_primary_complaint_display` will return the human readable value of `primary_complaint`

**Review by Jkrzy — CHANGES_REQUESTED** [review]
Looks amazing, thanks for tackling this one @cantsin!

 Noted a small few tweaks we could make before merging this into develop.

File: crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html
Comment (cantsin) [reply crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html:3]: Welp! Yes. I need to go back and re-do this bit. Thanks for the catch.

Comment (cantsin) [reply crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html]: Updated ... :sweat_smile:

File: crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html
Comment (Jkrzy) [inline crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html:11]: Ooo, hard coding `100` here results in us rendering 100 reports on the page _anytime_ `print_reports >= 100`. 
With the current UI, a user can only actually print those 100 reports if they've selected -- passed 100 IDs in their `GET`, or `print_reports.count() == 100`.

There's an opportunity here to save a bit of work and limit what we're rendering in the template to reports which can be printed by the end user. I'd call this an optimization and we don't need to block merging this whole PR on it but wanted to mention!

**Review by Jkrzy — APPROVED** [review]
Overall looks :amaze: thanks @cantsin! I left one comment inline above re: a potential area we could optimize and do a bit less rendering.

File: crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html
Comment (cantsin) [reply crt_portal/cts_forms/templates/forms/complaint_view/actions/bulk_print.html:11]: Good catch. I'm playing with this idea over at https://github.com/usdoj-crt/crt-portal/pull/739

**Pull request merged**
